### DISCUSSION TITLE
TypeError: r.inherits is not a function

### DISCUSSION THREAD

**Discussion by windkind88** [webpack/webpack#16987] [Q&A]
Dear developers: 
I am in the **pngjs** project,   and the error is reported after packaging: **TypeError: r.inherits is not a function**, what is the reason? How to solve it?

**Comment by windkind88** [comment]
https://github.com/pngjs/pngjs
The project references this npm package, and the packaging keeps failing T T

**Comment by alexander-akait** [comment]
Sorry, please clarify your issue
